backend/aethel_db/filters.py

```diff
@@ -0,0 +1,26 @@
+from aethel.frontend import Sample
+from src.aethel.scripts.search import Query, Sample
```

Comment:
I'm not sure how importing from `src` works for you (it doesn't for me), but it shouldn't be used like that[1]. We might have to ask Kokos to move the Query and Sample classes somewhere that's importable.

[1] The fact that the code for imported packages is found under `src/` is an implementation detail in some cases of using pip. I think it happens when you install a package with `pip -e`, but I'm not sure.